src/Client.cpp

```diff
@@ -1,21 +1,64 @@
 #include <iostream>
 #include <boost/array.hpp>
 #include <boost/asio.hpp>
+#include <tuple>
+#include <fstream>
+#include <stdio.h>
```

Comment:
Why is this needed?

Reply:
tuple was not needed, string and fstreme are.